IBPSA/BoundaryConditions/WeatherData/ReaderTMY3.mo

```diff
@@ -962,628 +962,229 @@ equation
           smooth=Smooth.Bezier,
```

Comment:
@AnaConstantin : Can you please revert these changes in `ReaderTMY3`. I don't think they are needed, and we don't want all library developers to have them review when they merge them into their library unless there is a real change that is needed.

Reply:
@mwetter sure. It's Dymola doing some cleaning up again. I'll double check next time.

Reply:
Thank you. Done! It should now show only the additional documentation I wrote.